.github/workflows/require-pr-label.yml

```
@@ -11,6 +11,7 @@ permissions:
 jobs:
   label:
     name: DO-NOT-MERGE / unresolved review
+    if: github.repository == 'python/cpython'
```

We use this in `stale.yml`, easier to copy between repos:

```suggestion
    if: github.repository_owner == 'python'
```